packages/api/src/beacon/routes/beacon/rewards.ts

```diff
@@ -89,12 +137,14 @@ export type Api = {
  */
 export const routesData: RoutesData<Api> = {
   getBlockRewards: {url: "/eth/v1/beacon/rewards/blocks/{block_id}", method: "GET"},
+  getAttestationsRewards: {url: "/eth/v1/beacon/rewards/attestations/{epoch}", method: "POST"},
   getSyncCommitteeRewards: {url: "/eth/v1/beacon/rewards/sync_committee/{block_id}", method: "POST"},
```

Comment:
> General question, why are sync committee rewards per block and not epoch?

From a user perspective, if I know my validator is chosen to be in the sync committee, I know that I will receive sync committee reward for every slot in every epoch during this period (256 epoch or 27hrs). So the smallest time unit for sync committee reward will be 1 slot. 

For attestations, I know I will be attesting for exactly one slot in every epoch. If this endpoint takes in a block_id instead of epoch, I will first need to check which slot in this epoch I attested to. So passing in an epoch value + my validator id, I can expect to see the attestation reward of my validator for that one slot that I attested to.